conans/client/conan_api.py

```diff
@@ -985,6 +986,15 @@ def remote_update_pref(self, package_reference, remote_name):
     def remote_clean(self):
         return self._cache.registry.clear()
 
+    @api_method
+    def remove_system_reqs(self, ref):
+        self._cache.remove_system_reqs(ref)
```

Comment:
Better to call the package layout directly. We are trying to promote layouts over functions in the `Cache` class.

```suggestion
        self._cache.package_layout(ref).remove_system_reqs()
```

Reply:
Thanks for the info!